### PR TITLE
ci: update nodejs version to v10.16.0

### PR DESCRIPTION
**Pull request by devversion**
* See individual commits for more details.

### CODE REVIEW

**Comment by alexeagle** [comment]
Can we fix the missing pinning on selenium-webdriver?

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "license": "MIT",
   "engines": {
     "node": ">=10.9.0 <11.0.0",
```
Comment (alexeagle) [inline package.json]: should the minimum node be advanced here? or we don't care since Bazel will download its own?

**Review by alexeagle — APPROVED** [review]
super-fishy that our CI breaks when someone else releases... there's a bug somewhere in our pinning

**Comment by devversion** [comment]
@alexeagle Yeah I was surprised as well. Unfortunately we can't lock this easily because the integration test depends on `ng new` which generates a `package.json` file that is not locking dependencies. Meaning that `protractor` brings in the most recent minor version of `selenium-webdriver`.

File: package.json (hunk 1 above)
Comment (devversion) [reply package.json]: I don't think we care too much about it. We only bumped it at some point to ensure that caching works or that network errors are reported properly.

**Comment by josephperrott** [comment]
@devversion Can we open an issue in [angular/angular-cli](https://github.com/angular/angular-cli) to track it?  Because its probably not good to be unpinned on their end either
.

**Comment by devversion** [comment]
@josephperrott I'll check if we already have one or create one if that's not the case.

**Review by IgorMinar — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
-    "yarn": ">=1.12.1 <=1.14.0"
+    "yarn": ">=1.12.1 <=1.16.0"
```
Comment (gkalpak) [inline package.json]: Why this change?
Aren't we still using 1.13.0 on CI?
  If so, is it just to make it easier for people to use the latest locally?
    If so, shouldn't the same change be made in `aio/package.json`?

<sub>(So many questions... 😛)</sub>

**Pull request closed without merging**